TODO.md

```
@@ -1,30 +1,65 @@
-  - [x] Bump version to 1.0.0.
-
-## Future / v1.1+
-
-- [ ] **GitHub Action**: A first-party action to run tokmd in CI.
-- [ ] **Diff Mode**: `tokmd diff <ref1> <ref2>` to see changes in stats.
-- [x] **Binary Releases**: Pre-built binaries attached to GitHub Releases (via `.github/workflows/release.yml`).
+## Completed
+
+### v1.0.0 — Production Readiness
+- [x] Formal JSON schema for `lang`, `module`, and `export`
+- [x] `schema_version`, `tool`, `inputs` metadata
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)